src/nomad_catalysis/schema_packages/catalysis.py

```diff
@@ -0,0 +1,378 @@
+class Preparation(ArchiveSection):
+    preparation_method = Quantity(
+        type=str,
+        shape=[],
+        description="""
+          Classification of the dominant preparation step
+          in the catalyst synthesis procedure.
+          """,
+        a_eln=dict(
+            component='EnumEditQuantity',
+            props=dict(
+                suggestions=[
+                    'precipitation',
+                    'hydrothermal',
+                    'flame spray pyrolysis',
+                    'impregnation',
+                    'calcination',
+                    'unknown',
+                ]
+            ),
+            links=['https://w3id.org/nfdi4cat/voc4cat_0007016'],
+        ),
+    )
+
+    preparation_entry_reference = Quantity(
+        type=Process,
+        shape=[],
+        description="""
+        A reference to the entry that contains the details of the preparation method.
+        """,
+        a_eln=dict(component='ReferenceEditQuantity'),
+    )
+
+    preparator = Quantity(
+        type=str,
+        shape=[],
+        description="""
+        The person or persons preparing the sample in the lab.
+        """,
+        a_eln=dict(component='EnumEditQuantity'),
+    )
+
+    preparing_institution = Quantity(
+        type=str,
+        shape=[],
+        description="""
+        institution at which the sample was prepared
+        """,
+        a_eln=dict(
+            component='EnumEditQuantity',
+            props=dict(
+                suggestions=[
+                    'Fritz-Haber-Institut Berlin / Abteilung AC',
+                    'Fritz-Haber-Institut Berlin / ISC',
+                ]
+            ),
+        ),
+    )
+
+    def normalize(self, archive, logger):
+        super().normalize(archive, logger)
+
+
+class SurfaceArea(ArchiveSection):
+    m_def = Section(
+        label_quantity='method_surface_area_determination',
+        a_eln=ELNAnnotation(label='Surface Area'),
+    )
+
+    surface_area = Quantity(
+        type=np.float64,
+        unit=('m**2/g'),
+        a_eln=dict(
+            component='NumberEditQuantity',
+            defaultDisplayUnit='m**2/g',
+            links=['https://w3id.org/nfdi4cat/voc4cat_0000013'],
+        ),
+    )
+
+    method_surface_area_determination = Quantity(
+        type=str,
+        shape=[],
+        description="""
+          A description of the method used to measure the surface area of the sample.
+          """,
+        a_eln=dict(
+            component='EnumEditQuantity',
+            props=dict(
+                suggestions=[
+                    'BET',
+                    'H2-TPD',
+                    'N2O-RFC',
+                    'Fourier Transform Infrared Spectroscopy (FTIR)' ' of adsorbates',
+                    'unknown',
+                ]
+            ),
+        ),
+    )
+
+    dispersion = Quantity(
+        type=np.float64,
+        shape=[],
+        description="""
+        The dispersion of the catalyst in %.
```

Comment:
Consider changing it to fraction.

Reply:
looking into a better definition, I found that IUPAC dicourages the use of the term dispersion for this application, but suggests the term "percentage exposed", which does not work if we want to get away from %. see https://goldbook.iupac.org/terms/view/P04485